docs/getting_started/install_on_ibexa_cloud.md

```diff
@@ -83,3 +88,13 @@ git push -u <platform.sh-remote> master
 !!! note
 
     `master` is the Platform.sh name for the production branch.
+
+!!! caution
+
+    If you want to use the [Platform.sh CLI](https://docs.platform.sh/development/cli.html), you have to set it up for Ibexa Cloud.
+    Concidere the following alias to use `ibexa_cloud` with the right set up instead of `platform`:
+    ```bash
+    alias ibexa_cloud="PLATFORMSH_CLI_SESSION_ID=ibexa_cloud \
+        PLATFORMSH_CLI_API_URL=https://api.cloud.ibexa.co \
+        platform"
```

Comment:
I don't see what the `PLATFORMSH_CLI_SESSION_ID` is for.

`PLATFORMSH_CLI_API_URL=https://api.cloud.ibexa.co platform project:list` works fine.

Reply:
> we recommend setting PLATFORMSH_CLI_SESSION_ID as well to ensure there is no conflict between the platform CLI session and ibexa CLI session, so that they can both work fine in parallel.